analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S1145_cs.html

```diff
@@ -1,36 +0,0 @@
-<p><code>if</code> statements with conditions that
```

Comment:
Why is this one deleted? This PR should only delete S4142 related stuff, or am I missing something?

Reply:
I've missed these files while deleting S4142 and found them now.